common/metrics.py

```
@@ -0,0 +1,209 @@
+        
+        # Update choices count
+        choices_count = stats.get('choices_count', 0)
+        cls.cache_choices_count.labels(model=model).set(choices_count)
+        
+        # Update Redis keys count
+        redis_keys = stats.get('redis_keys', {})
+        redis_keys_count = len(redis_keys) if redis_keys else 0
+        cls.cache_redis_keys_count.labels(model=model).set(redis_keys_count)
+
+    @classmethod
+    def time_cache_operation(cls, model: str, operation: str):
```

Ruff (0.12.2)

Missing return type annotation for classmethod `time_cache_operation`

(ANN206)